docs/micro-rdk/base/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Micro-RDK Base Models"
+linkTitle: "Base"
+weight: 30
+type: "docs"
+description: "Base support in the micro-rdk."
+images: ["/icons/components/base.svg"]
+tags: ["base", "components", "micro-rdk"]
```

Comment:
```suggestion
tags: ["base", "components", "micro-rdk"]
no_list: true
```
[opt] I think on all of these pages the cards at the bottom are redundant with the model table so would be good to remove? Marking as opt in case this was intentional

Reply:
Done!